### PR TITLE
Update prettier

### PR DESCRIPTION
**Pull request by trusktr**
~Do not merge yet, WIP.~

Alright, it's ready (feel free to modify as needed).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,14 +25,18 @@ module.exports = function(opt) {
         printWidth: 80,
         // Number of spaces it should use per tab
         tabWidth: 2,
-        // Use the flow parser instead of babylon
-        useFlowParser: true,
+        // Use tabs instead of spaces
+        useTabs: false,
+        // Remove semicolons
+        semi: true,
```
Comment (adamcohen) [inline index.js]: doesn't `semi: true` mean to *keep* semicolons?  If that's the case, the comment above should be updated to `// Keep semicolons`

**Comment by mlavina** [comment]
@trusktr @adamcohen @bhargavrpatel 

Any updates on this? I really want to use this plugin, but I need the latest gulp.

Thank you.

**Comment by trusktr** [comment]
@mlavina You could always just install the fork from this PR, straight from GitHub, using NPM. The changes are really simple, so you could also just fork and make the changes yourself too. :)

**Comment by bhargavrpatel** [comment]
Hi folks, apologies for the big delay. @trusktr Would you mind squashing your "cosmit" like commits into one?

**Comment by mlavina** [comment]
@trusktr I was about to just copy and paste this library into mine just because i wanted to use the latest prettier with gulp. However, since your PR is up I don't have to haha.

@bhargavrpatel Thanks for the quick response. Hope I can get a release of this with the latest prettier ASAP.

**Comment by jpoon** [comment]
Just came across this project and hit the same issue with it running an older version of prettier. 

There should be a "Squash and Merge" button (https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits).

**Comment by trusktr** [comment]
@bhargavrpatel Does the "Squash and Merge" button work? Or should I go ahead and do it manually?

**Comment by bhargavrpatel** [comment]
@trusktr apologies. I'll do a merge. Also a note that I am open to transferring ownership of this project to more active members of the Javascript community.

**Comment by TheDancingCode** [comment]
I wouldn't mind taking over the maintenance of this project.

**Comment by TheDancingCode** [comment]
If you need a token of my good intentions, I can submit a PR with one of the changes I would like to propose for this package.

**Comment by bhargavrpatel** [comment]
Go for it!

**Comment by bhargavrpatel** [comment]
@TheDancingCode Adding you as the repo admin now.

**Comment by TheDancingCode** [comment]
Thank you.

**Comment by bhargavrpatel** [comment]
@TheDancingCode Feel free to merge any PRs you have right now. I'll be pushing a short commit which notes that you are the new maintainer as well as thanking the previous collaborators in the README. 

Once again, thank you for all of your help folks:
@TheDancingCode 
@akella 
@trusktr

**Comment by TheDancingCode** [comment]
I can't merge the already opened PRs because they require a review and I can't review my own submissions.

**Comment by bhargavrpatel** [comment]
I've gone ahead and removed the requirement of a review for now. Since you
are added as a collaborator, feel free to push to master. You can decide
what works best for you.

On Thu, Apr 26, 2018 at 11:55 AM, Thomas Vantuycom <notifications@github.com

**Comment by TheDancingCode** [comment]
Alright. One more thing: could I get rights to publish to npm so that I can push new versions of this plugin?

**Comment by bhargavrpatel** [comment]
Transferred the ownership of this package to @TheDancingCode as of today.

**Pull request merged**